### DISCUSSION TITLE
Cudos dashboard version upgrade

### DISCUSSION THREAD

**Discussion by panchalnimesh** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#531] [General]
There is new release version 4.77.0 for cudos dashboard, will it update version automatically

**Comment by iakov-gan** [comment]
No updates are not automatic. Please follow this doc: 
https://wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/postdeploymentsteps/update/
